cmd/microindex/root/command.go

```diff
@@ -41,11 +41,12 @@ func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 
 func (c *Command) Run(args []string) error {
 	defer c.cli.Cleanup()
-	if ok, err := c.cli.Init(); !ok {
-		return err
-	}
+	err := c.cli.Init()
 	if len(args) == 0 {
 		return MicroIndex.Exec(c, []string{"help"})
 	}
+	if err != nil {
```

Comment:
drop line 44 and do
```suggestion
	if err := c.cli.Init(); err != nil {
```

Reply:
If you do this, then you'll get the help screen when you run `microindex -version`.

Reply:
It's a little fragile.  I think I'll leave it as is until we come up with something better.